widgetastic_manageiq/__init__.py

```diff
@@ -3290,6 +3290,18 @@ def name(self):
         else:
             return getattr(self, "_name", None)
 
+    def ensure_checked(self):
+        if not self.is_checked:
```

Comment:
Can you use `wait_for` here?

Reply:
I don't believe wait_for is needed here. I have tested this fairly well and the check functionality is essentially instantaneous.